var/spack/repos/builtin/packages/intel-oneapi-tbb/package.py

```diff
@@ -27,3 +27,38 @@ def __init__(self, spec):
                             releases=releases,
                             url_name='tbb_oneapi')
         super(IntelOneapiTbb, self).__init__(spec)
+
+#    def __init__(self, spec):
+#        self.component_info(dir_name='mkl',
+#                            components='intel.oneapi.lin.mkl.devel',
+#                            releases=releases,
+#                            url_name='onemkl')
+#        super(IntelOneapiMkl, self).__init__(spec)
+# FTW: Need to work in paths for  /soft/spack/opt/spack/linux-opensuse_leap15-cascadelake/oneapi-2021.1/intel-oneapi-mkl-2021.1.1-n2zoxk4l5w62slfcyp3sdirrrxp77bko/mkl/latest/../2021.1.1/lib/intel64 (all the mkl stuff) plus tbb: /soft/spack/opt/spack/linux-opensuse_leap15-cascadelake/gcc-10.1.0/intel-oneapi-compilers-2021.1.0-wgzjfktx5hsx75gghtd6k2qxad5vnot2/tbb/2021.1.1/lib/intel64/gcc4.8
+    def _join_prefix(self, path):
+        return join_path(self.prefix, 'tbb', 'latest', path)
+
+    def _ld_library_path(self):
```

Comment:
It looks like I should move all this down into the base class because it is used by multiple packages. Proceed with your commit and I can restructure later.

Reply:
_join_prefix can be refactored to the superclass; the _library ones use different lists, so would need to be parameterized for that if we move it.